cfn-resources/util/validator/validator.go

```diff
@@ -18,22 +17,7 @@ type ValidatorDefinition interface {
 	GetListFields() []string
 }
 
-func ValidateModel(event constants.Event, def ValidatorDefinition, model interface{}) *handler.ProgressEvent {
-
-	fields := []string{}
-
-	switch event {
-	case constants.Create:
-		fields = def.GetCreateFields()
-	case constants.Read:
-		fields = def.GetReadFields()
-	case constants.Update:
-		fields = def.GetUpdateFields()
-	case constants.Delete:
-		fields = def.GetDeleteFields()
-	default:
-		fields = def.GetListFields()
-	}
```

Comment:
Just wondering what the reason is for this refactor?

Reply:
To remove the Validator class from every module([cfn-resources/**<<module_name>>/cmd/validation/validate.go**]  and reuse the Validator class available in util package